test/Npgsql.PluginTests/JsonNetTests.cs

```diff
@@ -54,6 +56,40 @@ public void DeserializeFailure()
             }
         }
 
+        [Test, IssueLink("https://github.com/npgsql/npgsql/issues/3085")]
+        public void RoundtripStringTypes()
+        {
+            var expected1 = @"{""Key"": ""Value""}";
```

Comment:
I think that `"{\"p\": 1}"` can be used for all cases:

```csharp
var expected = "{\"p\":1}";
var expected1 = expected.ToCharArray();
var expected2 = new ArraySegment<char>(expected1);
var expected3 = Encoding.ASCII.GetBytes(expected);
```

Reply:
Good pickup, it will clean up nicely.